analyzers/tests/SonarAnalyzer.UnitTest/Extensions/SyntaxNodeExtensionsTest.cs

```diff
@@ -172,20 +172,68 @@ public void ArrowExpressionBody_WithNotExpectedNode_ReturnsNull()
             ExtensionsCS.GetDeclarationTypeName(SyntaxFactory.StructDeclaration("MyStruct")).Should().Be("struct");
 
         [TestMethod]
-        public void CreateCfg_MethodBody_ReturnsCfg_CS()
+        public void CreateCfg_MethodDeclaration_ReturnsCfg_CS()
         {
-            const string code = @"
-public class Sample
-{
-    public void Main()
-    {
-        var x = 42;
-    }
-}";
+            const string code = """
+                public class Sample
+                {
+                    public void Main()
+                    {
+                        var x = 42;
+                    }
+                }
+                """;
             var (tree, semanticModel) = TestHelper.CompileCS(code);
             var node = tree.Single<SyntaxCS.MethodDeclarationSyntax>();
 
-            ExtensionsCS.CreateCfg(node.Body, semanticModel, default).Should().NotBeNull();
+            ExtensionsCS.CreateCfg(node, semanticModel, default).Should().NotBeNull();
+        }
+
+        [TestMethod]
+        public void CreateCfg_PropertyDeclartion_ReturnsCfg_CS()
+        {
+            const string code = """
+                public class Sample
+                {
+                    int field;
+                    public int Property => field + 42;
```

Comment:
To remove scaffolding noise
```suggestion
                    public int Property => 42;
```